__tests__/app.test.js

```diff
@@ -124,4 +124,73 @@ describe("app", () => {
       return request(app).get("/api/articles").expect(500);
     });
   });
+
+  describe("GET /api/articles/:article_id", () => {
+    it("responds with a status 200 if successful", () => {
+      return request(app).get("/api/articles/1").expect(200);
+    });
+    it("responds with only one article", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((res) => {
+          let article = res.body.article;
+          expect(article.length).toBe(1);
+        });
+    });
+    it("responds with an article object with the correct properties", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((res) => {
+          let article = res.body.article;
+          expect(article).toEqual(
+            expect.arrayContaining([
+              expect.objectContaining({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+              }),
+            ])
+          );
+        });
+    });
+    it("handles variety of ids!", () => {
+      return request(app)
+        .get("/api/articles/2")
+        .expect(200)
+        .then(() => {
+          return request(app).get("/api/articles/12").expect(200);
+        });
+    });
+    it("responds with an error code of 404 if the article cannot be found", () => {
+      return request(app).get("/api/articles/200").expect(404);
```

Comment:
it might be nice to send back a message as well as an error code and test for this also